iceoryx_examples/icedelivery/README.md

```diff
@@ -63,38 +50,40 @@ struct RadarObject
 ```
 
 It is included by:
+
 ```cpp
 #include "topic_data.hpp"
 ```
 
 For the communication with RouDi a runtime object is created. The parameter of the method `initRuntime()` contains a
 unique string identifier for this publisher.
+
 ```cpp
-iox::runtime::PoshRuntime::initRuntime("iox-ex-publisher");
+constexpr char APP_NAME[] = "iox-cpp-publisher-untyped";
+iox::runtime::PoshRuntime::initRuntime(APP_NAME);
 ```
 
 Now that RouDi knows our publisher application is existing, let's create a publisher instance for our charming struct
 to everyone:
+
 ```cpp
-iox::popo::UntypedPublisher untypedPublisher({"Radar", "FrontLeft", "Object"});
+iox::popo::UntypedPublisher publisher({"Radar", "FrontLeft", "Object"});
 ```
 
-The strings inside the first parameter of the constructor of `iox::popo::Publisher` are of the type
-`capro::ServiceDescription`. `capro` stands for **ca**nionical **pro**tocol and is used to abstract different
-[SoA](https://en.wikipedia.org/wiki/Service-oriented_architecture) protocols. `Radar` is the service name, `FrontLeft`
-an instance of the service `Radar` and the third string the specific event `Object` of the instance.
-In iceoryx a publisher and a subscriber only match if all the three IDs match.
-
 Now comes the work mode. Data needs to be created. But hang on.. we need memory first! Let's reserve a memory chunk
 which fits our RadarObject struct
+
 ```cpp
-auto result = untypedPublisher.loan(sizeof(RadarObject));
+auto result = publisher.loan(sizeof(RadarObject));
 ```
 
+#### #1 Loan and emplace
```

Comment:
This is not the way we endorse! We suggest to use always the functional approach and offer the traditional way only as an alternative. This should be reflected here so that the functional approach comes first and this appraoch is shown as an alternative.

Reply:
Done